app/jobs/reports/sp_user_counts_report.rb

```diff
@@ -28,7 +28,7 @@ def perform(_date)
     def track_report_data_events(user_counts)
       user_counts.each do |hash|
         track_report_data_event(
-          Analytics::REPORT_SP_USER_COUNTS,
+          'Report SP User Counts',
```

Comment:
this only goes to `reports.log` so it never went to `events.log` right?

Reply:
correct

Reply:
I guess eventually we have to create a separate documentation section for the reports.log